junit5/src/main/java/sunstone/core/AzureArmTemplateCloudDeploymentManager.java

```diff
@@ -107,33 +101,24 @@ private String parametersFromMap(String template, Map<String, String> parameters
         return r;
     }
 
-
-
-    @Override
     public void undeploy(String rgName) {
         ResourceGroups rgs = armManager.resourceGroups();
         if (rgs.contain(rgName)) {
             rgs.deleteByName(rgName);
         }
+        usedRG.remove(rgName);
     }
 
-    @Override
-    public void register(String id) {
-        usedRG.add(id);
-    }
-
-    public void close() {
-        // nothing to close
+    public void register(String group) {
```

Comment:
Is `register()` method needed or could it be removed?

Reply:
Good point. it is not really needed after I removed the interface. I will delete it.